src/Symfony/Component/Cache/Traits/RedisCluster6Proxy.php

```diff
@@ -0,0 +1,979 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Traits;
+
+use Symfony\Component\VarExporter\LazyProxyTrait;
+
+/**
+ * @internal
+ */
+class RedisCluster6Proxy extends \RedisCluster
```

Comment:
Wouldn't those committed proxies break if Redis has a new method in a minor version ?

Reply:
They would. That's why I added a test case to ensure we're in sync.

Reply:
but then, this means that all *existing* releases of Symfony become broken when ext-redis releases a new version, until we release a version with the new methods. I find that quite bad DX for our users.

Reply:
They are not really broken until you start using the new methods. I think this is acceptable since ppl using the new methods can wait a new release. The window should be pretty short.